File: olymp/2022/main_d.py
file = open("input.txt", "r").readlines()

array = list()

for line in file:
  array.append(line.split())

k = int(array[0][0])
m = int(array[1][0])
x = int(array[2][0])
y = int(array[3][0])

kolya = 0
misha = 0

i = x
while i <= y:
	if i % k == 0: 
		kolya = kolya + 1
	if i % m == 0:
		misha = misha + 1
		
	i = i + 1

print (kolya - misha)

File: olymp/2022/main_d.cpp
#include "iostream"
#include <fstream>

int main() {
    int sum = 0;
    int x;
    std::ifstream inFile;

    inFile.open("./input.txt");
    if (!inFile) {
        std::cout << "Unable to open file";
        exit(1); // terminate with error
    }

    while (inFile >> x) {
        sum = sum + x;
    }

    inFile.close();
    std::cout << "Sum = " << sum << std::endl;
    return 0;
}
